pkg/domain/stateDiffConfig.go

```diff
@@ -49,6 +52,20 @@ const (
 	ConfigActionRemove ConfigAction = "remove"
 )
 
+func determineConfigDiff(
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>func `determineConfigDiff` is unused